sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/ResolveDefaultColumnsUtil.scala

```diff
@@ -241,4 +242,36 @@ object ResolveDefaultColumns {
       }
     }
   }
+
+  /**
+   * This is an Analyzer for processing default column values using built-in functions only.
+   */
+  object DefaultColumnAnalyzer extends Analyzer(
+    new CatalogManager(BuiltInFunctionCatalog, BuiltInFunctionCatalog.v1Catalog)) {
+    override def resolver: Resolver = caseSensitiveResolution
```

Comment:
Why use `caseSensitiveResolution` here? I saw this in `SimpleAnalyzer` though. But I think we don't need to overwrite it. The default `conf.resolver` looks more reasonable.

Reply:
Good point, I updated this to use the default `conf.resolver` instead for simplicity.